core/src/main/scala/kafka/coordinator/GroupMetadataManager.scala

```diff
@@ -538,48 +551,57 @@ class GroupMetadataManager(val brokerId: Int,
         offsetAndMetadata.expireTimestamp < startMs
       }
 
-      debug("Found %d expired offsets.".format(expiredOffsets.size))
+      deleteExpiredOffsets(expiredOffsets)
+    }
+
+    info("Removed %d expired offsets in %d milliseconds.".format(numExpiredOffsetsRemoved, SystemTime.milliseconds - startMs))
```

Comment:
nit: any reason why this shouldn't also use string interpolation as you did above?